### ISSUE TITLE
Multiple types on separate lines issue

### ISSUE THREAD

**Issue by adithyabsk** [terrencepreilly/darglint#19]
This docstring
```python
"""Test.

Args:
    input (:obj:`DataFrame <pandas.DataFrame>`, \
        :obj:`ndarray <numpy.ndarray>`, list): test

"""
```
Raises this issue: `S001 Syntax error: s Unable to parse word: expected TokenType.WORD but received TokenType.NEWLINE`

The line continuation is used to respect PEP-8 guidelines.

**Comment by adithyabsk** [comment]
I'm also unable to silence this issue using #noqa: S001 in the docstring

**Comment by terrencepreilly** [comment]
Good catch.  I'm not surprised it's not silencing: the error is raised when the parser fails.  We should still be able to silence it, though.  Thanks!

**Comment by terrencepreilly** [comment]
Line continuation should be handled now for types descriptions.  Your example above won't work, but that's unrelated to line continuation.

I've also updated the reporter so that it checks for the appropriate `# noqa` statements when the parser fails.  So, if you add any of `# noqa: S001`, `# noqa` or `# noqa: *`, it should suppress this error message.

Resolved in v0.5.5
